.github/workflows/backend_dev.yml

```diff
@@ -62,7 +62,7 @@ jobs:
 #        run: python -m unittest discover -v
       # About: https://azure.github.io/AppService/2020/12/11/cicd-for-python-apps.html
       - name: Upload artifact for deployment jobs
-        uses: actions/upload-artifact@v2
+        uses: actions/upload-artifact@v4
```

Comment:
## The bug fix
These needed to be upgraded. Actions / deploys were [suddenly failing](https://github.com/jhu-bids/TermHub/actions/runs/10963128923).

Explanation: https://github.blog/changelog/2024-02-13-deprecation-notice-v1-and-v2-of-the-artifact-actions/